RecoMuon/GlobalTrackingTools/plugins/GlobalTrackQualityProducer.cc

```diff
@@ -285,11 +285,9 @@ std::pair<double,double> GlobalTrackQualityProducer::newChi2(Trajectory& muon) c
     }
   }
   
-  if (tkNdof < 6 ) tkChi2 = tkChi2; // or should I set it to  a large number ?
-  else tkChi2 /= (tkNdof-5.);
+  if (tkNdof > 5 ) {tkChi2 /= (tkNdof-5.); }
```

Comment:
is it really obvious that ```tkChi2 = tkChi2; // or should I set it to  a large number ?``` was not a typo?

Reply:
From 'blame' that line has been in the code for 10 years.

Reply:
https://github.com/cms-sw/cmssw/blame/master/RecoMuon/GlobalTrackingTools/plugins/GlobalTrackQualityProducer.cc#L288

Reply:
I "refreshed" my memory a bit.
This part was questionable to me back when it was proposed.
At least for historical reason, I propose to leave a comment here about the other values; smth like ```if (tkNdof < 6 ) the tkChi2 is used directly: should a large number or something else be used``` perhaps with a FIXME.

@drkovalskyi is anybody still using these IDs?